### PR TITLE
Remove validation of amount sign to maintain backward-compatibility

### PR DESCRIPTION
**Pull request by ritchiey**

### CODE REVIEW

File: lib/aba/batch.rb [hunk 1]
```diff
@@ -193,7 +194,7 @@ def batch_control_record
       # Net total
       # Max: 10
       # Char position: 21-30
-      output += net_total_amount.abs.to_s.rjust(10, "0")
+      output += (credit_total_amount - debit_total_amount).abs.to_s.rjust(10, "0")
```
Comment (bazlo) [inline lib/aba/batch.rb:197]: I don't think `abs` is required here anymore.
```suggestion
      output += (credit_total_amount - debit_total_amount).to_s.rjust(10, "0")
```

**Review by bazlo — APPROVED** [review]
LGTM

File: lib/aba/entry.rb [hunk 2]
```diff
@@ -7,11 +7,11 @@ def initialize(attrs = {})
...
-      (50..57).include?(transaction_code.to_i)
+      Validations::CREDIT_TRANSACTION_CODES.include?(transaction_code.to_i)
     end
 
     def debit?
-      transaction_code.to_i == 13
+      Validations::DEBIT_TRANSACTION_CODES.include?(transaction_code.to_i)
     end
-  end
+  end
```
Comment (igas) [inline lib/aba/entry.rb]:
```suggestion
  end
```

Comment (trev) [reply lib/aba/entry.rb]: Solid review @igas :)

File: lib/aba/batch.rb (hunk 1 above)
Comment (trev) [reply lib/aba/batch.rb:197]: Actually this has to stay in case the `debit_total_amount` is larger than the `credit_total_amount`

Comment (bazlo) [reply lib/aba/batch.rb:197]: Yes that makes sense, now you say it. 
Is there a spec to highlight the larger debit than credit total behaviour?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## v1.0.0, 12 July 2020
+
+### BREAKING CHANGES
+
+* Positive (`n`) and negative (`-n`) values are now treated the same.
+  e.g `5` and `-5` are both processed as `5`, without any signage.
+  To differentiate between a debit and credit, use the correct [Transaction Code](https://github.com/andrba/aba/blob/58446f5b0ef822e9792e9399b4af647319b13515/lib/aba/transaction.rb#L106-L112)
+* Removed default values for transactions records to avoid generation of potentially incorrect records. Safety first!
```
Comment (ritchiey) [inline CHANGELOG.md:8]: Nice! 🥇

**Pull request merged**